microcluster/app.go

```diff
@@ -40,8 +41,9 @@ type MicroCluster struct {
 type Args struct {
 	StateDir string
 
-	Client *client.Client
-	Proxy  func(*http.Request) (*url.URL, error)
+	Client      *client.Client
+	Proxy       func(*http.Request) (*url.URL, error)
+	ClientCache tls.ClientSessionCache
```

Comment:
Is there a specific use case to have the cache configurable from the outside?

Reply:
Not a specific one, but since microclusters can have varied uses I thought there might be cases where the default is too big/too small for individual use cases, particularly if the dqlite heartbeat interval has been altered as well.

Reply:
Can we pick the size of the cache dynamically based on the number of cluster members? I guess each cache entry will represent the TLS session with another cluster member?

Reply:
Hmm, we could do that but it would involve re-initializing the cache with each new joiner. Might have some performance implications, but it's a good note for testing the ideal setup.